source/documentors/concepts/doc_audiences.rst

```diff
@@ -3,11 +3,9 @@ Documentation Audiences
 
 Open edX documentation is organized by the following audiences:
 
-* **Educators**, who build online courses using Studio.
+* **Educators**, who build and/or run online courses using Studio.
```

Comment:
@jswope00 - is this the best way to describe educators? I feel "build" would be better as "author"